make/build_test.mk

```diff
@@ -48,7 +48,7 @@ $($(T)_TESTS): $($(T)_TEST_BIN_DIR)/%_runner$(PLATFORM_EXT): \
 	@echo "Building test $(notdir $@) for $(PLATFORM)"
 	@$(CC) $(CFLAGS) -Wl,-Map=$|/$(notdir $(@:%$(PLATFORM_EXT)=%.map)) $^ -o $@ -L$(STATIC_LIB_DIR) \
 		$(addprefix -l,$(APP_DEPS)) \
-		$(LDFLAGS) $(addprefix -I,$(INC_DIRS))
+		$(LDFLAGS) $(addprefix -I,$(INC_DIRS)) $(LDADD)
```

Comment:
@ckitagawa I fixed the issue.

Turns out it was the position of the `lrt` flag. I think you were adding them to ``make/build.mk`` instead of ``make/build_test.mk``? Either way, it works now.